afdsafdsafdsa

```diff
@@ -0,0 +1,212 @@
+[1mdiff --git a/events/api.py b/events/api.py[m
```

Comment:
Bloat?

Reply:
git diff probably did something to this. I'll fix it.